global.go

```diff
@@ -65,29 +65,32 @@ func newGlobalManager(conf BehaviorConfig, instance *V1Instance) *globalManager
 			Help: "The count of requests queued up for global broadcast.  This is only used for GetRateLimit requests using global behavior.",
 		}),
 	}
-	gm.runAsyncHits()
-	gm.runBroadcasts()
+	gm.toOwners()
```

Comment:
I see the confusion around the original names.  Would prefer a more `<Verb><Noun>` nomenclature.

How about:
- `gm.runAsyncHits()` -> `gm.runBatches()`

I think `runBroadcasts()` is appropriately named.

Thoughts?

Reply:
No, for me it's still confusing, because i'm missing out on _who_ is doing the action.

Reply:
It's implied the current peer is doing the action.  Both of these start a goroutine that process these actions.